components/hab/src/command/cli/setup.rs

```diff
@@ -146,6 +167,27 @@ pub fn start(ui: &mut UI, cache_path: &Path, analytics_path: &Path) -> Result<()
     } else {
         ui.para("Okay, maybe another time.")?;
     }
+    if cfg!(target_os = "windows") {
+        ui.heading("Habitat Binlink Path")?;
+        ui.para(
+            "The `hab` command-line tool can create binlinks for package binaries  \
+             in the 'PATH' when using the 'pkg binlink' or 'pkg install --binlink' \
+             commands. By default, Habitat will create these binlinks in the '/hab/bin' \
+             directory. This directory will always be included in your 'PATH' when \
+             inside a Studio environment. However, you will want this directory on your \
+             machine's persistent 'PATH' in order to access binlinks outside of a Studio.",
```

Comment:
the `ask_set_binlink_path` function below actually serves this purpose. The convention we have been following in this file is to explain the scenario in the `para` blocks and then ask directly in a `prompt_yes_no` question. So after this text the user is given a yes/no question: `Add binlink directory to PATH?`

Reply:
I see. So it would look like the following, right?

- Habitat Binlink Path
  - [this paragraph]
- Add binlink directory to PATH? [Yes/no/quit] (if type 'yes')
  - Enter your Binlink directory.
- Binlink directory: _

Reply:
Almost. If `yes` is entered, we add the default binlink directory `c:\hab\bin` to the path and output:
```
c:\hab\bin has been added to your path. You will need to open a new console window for this added entry to take effect.
```

Reply:
I see. Looks good! Thanks for explaining this to me 🤓